build/Linux+BSD/portable/copy-libs

```diff
@@ -93,6 +93,8 @@ getCrossPlatformDependencies() {
 
   copyQt plugins/sqldrivers/libqsqlite.so
 
+  copyQt plugins/xcbglintegrations/libqxcb-glx-integration.so
```

Comment:
I believe on Windows, while the symptomps are similrat, the reason is different, a non-OpenGL 2.0 capable graphics Card (seemlingly needed by Qt, and as such nothing MuseScore could do to fix it), and affected only one user or at least only very few?